Enterprise-CMCS/macpro-mdct-mfp
# State and territory-specific Initiatives - Update wp.json

### Description
Set up wp.json State and territory Initiatives page. 
https://qmacbis.atlassian.net/jira/software/c/projects/MDCT/boards/232?modal=detail&selectedIssue=MDCT-2814

This is how it displays with the json passed into the mock component. 

<img width="920" alt="Screenshot 2023-08-29 at 4 18 04 PM" src="https://github.com/Enterprise-CMCS/macpro-mdct-mfp/assets/14514294/b842c861-2d5b-4696-afbe-b7963f809471">


### Related ticket(s)
https://qmacbis.atlassian.net/jira/software/c/projects/MDCT/boards/232?modal=detail&selectedIssue=MDCT-2814

---
### How to test
I removed the mock component to prepare for the routes update. 
You can see the UI by going to the /standard path and replacing 
export const mockDrawerReportPageJson object with this
```
export const mockStandardReportPageJson = {
  name: "mock-route-1",

  path: "/mock/mock-route-1",

  pageType: "standard",

  verbiage: {
    intro: {
      section: "State & Territory Specific Initiatives: I. Define Initiative",

      subsection: "Person Centered Planning",

      info: [
        {
          type: "html",

          content:
            "Provide initiative description, including target populations and timeframe.",
        },
      ],

      editEntityButtonText: "Edit",

      enterReportText: "Enter Details",
    },
  },

  form: {
    id: "test-mock-form",

    fields: [
      {
        id: "target_population",

        type: "checkbox",

        validation: "checkbox",

        props: {
          label: "Target Population(s):",

          hint: "Select all that apply. Population you've added as 'Other' in the Transition Benchmarks section will appear here.",

          choices: [
            {
              id: "2iuXO7C6nk6cuP9JXbdd2w",

              label: "Older adults",
            },

            {
              id: "vmlIjQAe9kyz4FbtxBZINA",

              label: "Individuals with physical disabilities (PD)",
            },

            {
              id: "Vg8erh64Tk2nKd5olVwM9w",

              label:
                "Individuals with intellectual and developmental disabilities(MH/SUD)",
            },

            {
              id: "azz5rhd8V0GK27fIXaYSmw",

              label: "Other: {HIV/AIDS}",
            },

            {
              id: "OLmKdPAEI0WnbSV1sVccVw",

              label: "Other third-party vendor",
            },

            {
              id: "Gxk89QOgQkmMTaNHH9WznQ",

              label: "Other, specify",

              children: [
                {
                  id: "state_encounterDataValidationEntity-otherText",

                  type: "textarea",

                  validation: {
                    type: "text",

                    nested: true,

                    parentFieldName: "state_encounterDataValidationEntity",
                  },
                },
              ],
            },
          ],
        },
      },

      {
        id: "mock-date-field",

        type: "date",

        validation: "date",

        props: {
          label: "Start Date",

          hint: "Enter projected start month/year for future initiatives or enter past start month/year for initiatives in the process.",
        },
      },

      {
        id: "mock-date-field",

        type: "date",

        validation: "date",

        props: {
          label: "End Date",

          hint: "Enter projected end date or “N/A” if the initiative will be ongoing without a set point.",
        },
      },
    ],
  },
};

```



### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
